### DISCUSSION TITLE
Toolips using single div reference

### DISCUSSION THREAD

**Discussion by jsheely** [mantinedev/mantine#7058] [Feature requests]
@rtivital What are your thoughts against updating <Tooltip /> to use a single portal div that updates and positions it's content.

Right now creating tooltips generates a unique div in the DOM at the bottom of the body for each unique component reference. 

In a perfect world we think it would make sense to have a single portal div that never is removed that simply moves into position and changes its content. 

Do you see any downsides to this strategy? If not I will update the component

**Comment by rtivital** [comment]
Portal component is used in all Mantine components: Modal, Popover, Tooltip, etc. If this change is implemented, the Portal component stacking context (z-index) will be the same for all components.
